load_testing/main.py

```diff
@@ -0,0 +1,50 @@
+"""
+This script loads a config file, runs multiple locust load tests, plots and writes
+results to file. Functionality based on: github.com/mohsenSy/locust-tests-runner/
+blob/master/locust_run_tests.py
+"""
+
+import argparse
+import json
+
+from custom_load_testing import reporting, testing
+
+
+def parse_args() -> argparse.Namespace:
+    """Parses arguments for the script."""
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-c",
+        "--config",
+        action="store",
+        default="configs/1_200_ramp_config.json",
+        help="JSON file containing configs for tests",
+    )
+    parser.add_argument(
+        "-o",
+        "--output",
+        action="store",
+        default="outputs",
+        help="Folder to store outputs",
+    )
+    parser.add_argument(
+        "--analyze-results-only",
+        action="store_true",
+        help="If this tag is included, analyze previously-run test output results only",
+    )
+    args = parser.parse_args()
```

Comment:
this is a bit confusing, i dont remember what I meant when i wrote this haha could you add these available parameters to the README too?

Reply:
@amiraliemami I don't actually think this is necessary